src/mappings/mappingHandlers.ts

```diff
@@ -0,0 +1,21 @@
+import assert from "assert";
+import { SubstrateEvent } from "@subql/types";
+
+import { PureProxy } from "../types";
+
+export async function handleEvent(event: SubstrateEvent): Promise<void> {
+  const {
+    event: {
+      data: [accountId, proxyAccountId, proxyType, delay],
+    },
+  } = event;
+
+  const pureProxy = PureProxy.create({
+    id: accountId.toHex(),
+    proxyAccountId: proxyAccountId.toHex(),
+    proxyType: proxyType.toString(),
```

Comment:
is it ok that we use it without mapping?
I would be better to fall down in indexing, then on clients side

Reply:
@stepanLav  What mapping do you mean? Just curious 😉